cves/CVE-2015-0219.yml

```diff
@@ -198,8 +219,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: true
+  note: This feature affects the development server so it must use IPC.
```

Comment:
Could you discuss a little further about this? Is there an active thread or process that's relaying information in the server?